setup.py

```diff
@@ -23,26 +23,42 @@
 
 packages = [
     "volumina",
+    "volumina._testing",
     "volumina.pixelpipeline",
+    "volumina.pixelpipeline.datasources",
+    "volumina.pixelpipeline.imagesources",
     "volumina.skeletons",
-    "volumina.widgets",
+    "volumina.tiling",
     "volumina.utility",
     "volumina.view3d",
-    "volumina._testing",
+    "volumina.widgets",
 ]
 
 package_data = {
     "volumina.widgets": ["*.ui", "ui/*.ui"],
     "volumina": ["*.ui"],
-    "volumina._testing": ["*.tif", "*.png", "lena.npy", "rgba129x104.npy"],
+    "volumina._testing": ["*.tif", "*.png", "2d_cells_apoptotic_1channel.npy", "rgba129x104.npy"],
 }
 
 setup(
```

Comment:
Most of the package metadata from `setup.py` can be moved to [declarative `setup.cfg`](https://setuptools.readthedocs.io/en/latest/userguide/declarative_config.html).

Reply:
it's so short, easily fits my screen. I'd rather consolidate setup.cfg with pyproject.toml, and leave setup.py as it is.